internal/backends/winit/event_loop.rs

```diff
@@ -63,7 +56,7 @@ impl NotRunningEventLoop {
             builder.with_any_thread(true);
         }
 
-        let instance = builder.build();
+        let instance = builder.build().expect("FIXME: handle error");
```

Comment:
Ah yes, this can finally fail. I think that this should go through the backend initialisation, i.e. `new()` should return an error and we should initialize `MAYBE_LOOP_INSTANCE` in the backend init and return an error. AFAIK the build fails for example if there's no `DISPLAY` or `WAYLAND_DISPLAY`, and in that case the entire backend init should fail, so that we can fall back to say linuxkms.